src/test/resources/application-test.properties

```diff
@@ -0,0 +1,11 @@
+# H2 Database Configuration for Testing
```

Comment:
tests are not working like i told you earlier, please debug it i told you has something to do with this file